photopicker/src/main/java/com/unsplash/pickerandroid/photopicker/data/UnsplashUser.kt

```diff
@@ -13,7 +13,7 @@ data class UnsplashUser(
     val location: String?,
     val total_likes: Int,
     val total_photos: Int,
-    val total_collection: Int,
+    val total_collection: Int?,
```

Comment:
why is this becoming nullable?

Reply:
Because the Unsplash API can return null for this field.